src/main/kotlin/no/nav/familie/tilbake/api/dto/ManuellBrevmottakerRequestDto.kt

```diff
@@ -0,0 +1,44 @@
+package no.nav.familie.tilbake.api.dto
+
+import no.nav.familie.kontrakter.felles.tilbakekreving.ManuellAdresseInfo
+import no.nav.familie.kontrakter.felles.tilbakekreving.Vergetype
+import no.nav.familie.tilbake.dokumentbestilling.manuell.brevmottaker.domene.MottakerType
+import java.util.UUID
+import javax.validation.constraints.Pattern
+
+data class ManuellBrevmottakerRequestDto(
+    val type: MottakerType,
+    val navn: String,
+    val manuellAdresseInfo: ManuellAdresseInfo? = null,
+    @field:Pattern(regexp = "(^$|.{9})", message = "Organisasjonsnummer er ikke riktig")
+    val organisasjonsnummer: String? = null,
+    @field:Pattern(regexp = "(^$|.{11})", message = "PersonIdent er ikke riktig")
+    val personIdent: String? = null,
+    val vergetype: Vergetype? = null
+) {
+    init {
+        check(!(manuellAdresseInfo == null && personIdent.isNullOrBlank() && organisasjonsnummer.isNullOrBlank())) {
+            "Må ha enten manuellAdresse, personIdent eller et organisasjonsnummer."
+        }
+        if (type == MottakerType.VERGE || type == MottakerType.FULLMEKTIG) requireNotNull(vergetype)
+    }
+}
+
+data class ManuellBrevmottakerResponsDto(
+    val id: UUID,
+    val type: MottakerType,
+    val navn: String,
+    val manuellAdresseInfo: ManuellAdresseInfo? = null,
+    @field:Pattern(regexp = "(^$|.{9})", message = "Organisasjonsnummer er ikke riktig")
+    val organisasjonsnummer: String? = null,
+    @field:Pattern(regexp = "(^$|.{11})", message = "PersonIdent er ikke riktig")
+    val personIdent: String? = null,
+    val vergetype: Vergetype? = null
+) {
+    init {
+        check(!(manuellAdresseInfo == null && personIdent.isNullOrBlank() && organisasjonsnummer.isNullOrBlank())) {
+            "Må ha enten manuellAdresse, personIdent eller et organisasjonsnummer."
+        }
+        if (type == MottakerType.VERGE || type == MottakerType.FULLMEKTIG) requireNotNull(vergetype)
+    }
```

Comment:
Kunne potensielt også gjenbrukt mer fra Brevmottaker i familie-kontrakter, med f.eks noe sånt:
```suggestion
    val id: UUID,
    val brevmottaker: Brevmottaker
```
Men vil ikke fungere rett ut av boksen uten å oppdatere ManuellBrevmottakerMapper i tillegg 🤔